tests/integration/s3/test_connect_to_region.py

```diff
@@ -57,6 +57,11 @@ def testSuccessWithDefaultUSEast1(self):
         self.assertEquals('s3.amazonaws.com', connection.host)
         self.assertIsInstance(connection, S3Connection)
 
+    def testSuccessWithDefaultEUCentral1(self):
```

Comment:
This test doesn't exercise the new S3 location constant you've added (i.e this test passes without your changes to s3/connection.py).

It would be great to have a test that verifies we can use the new constant to create a bucket in `eu-central-1`.